server/routes/people.ts

```diff
@@ -21,7 +21,7 @@ export default function routes(controllers: Controllers, router: Router, service
   })
 
   get(peoplePaths.timeline.find.pattern, timelineController.find(), { auditEvent: 'SEARCH_FOR_TIMELINE' })
-  post(peoplePaths.timeline.find.pattern, timelineController.show(), {
+  get(peoplePaths.timeline.show.pattern, timelineController.show(), {
```

Comment:
@pezholio are these verbs as you'd expect? `find` for the screen the user enters the CRN, `show` for the timeline screen itself?

Reply:
Yeah, I think that makes sense. I'm getting more and more agnostic about "REST-y" verbs as I go on, so I think whatever seems sensible is fine 👍

Reply:
What WOULD be great though would be if we could have a canonical URL for the timeline, so rather than having:

```
people/timeline/show?crn=ABC123
```

as the sharable / bookmarkable URL, we had:

```
people/CRN123/timeline
```

I guess to do that we'd POST to the `find` endpoint, if we find the user we redirect to the canonical URL, otherwise we return an error.

Reply:
I think it's mainly for "correctness" and ensure's there's a canonical URL that people can refer to. Working with Tim Berners-Lee has rubbed off on me! It's not a major thing

Reply:
I think in the interests of getting this in before the demo I think it might be best to merge and come back to it

Reply:
No, I totally agree 👍